docs/workshops/kubecon_azure_track.md

```diff
@@ -0,0 +1,114 @@
+Table of contents:
+- [Overview](#overview)
+- [Basic Cheet Sheet](#basic-cheet-sheet)
+- [Adjustments](#adjustments)
+    - [Step: Install Karpenter](#step-install-karpenter)
+    - [Step: Basic NodePool](#step-basic-nodepool)
+        - [Step: Scaling Application](#step-scaling-application)
+        - [Step: Limit Resources](#step-limit-resources)
+        - [Step: Disruption](#step-disruption)
+        - [Step: Drift](#step-drift)
+        - [Step: RightSizing](#step-rightsizing)
+    - [Step: Consolidation](#step-consolidation)
+        - [Step: Single Node Consolidation](#step-single-node-consolidation)
+        - [Step: Multi Node Consolidation](#step-multi-node-consolidation)
+
+## Overview
+
+This document details the steps to follow the KubeCon workshop using an AKS cluster in Azure.
+
+To follow along using this workshop, simply go through the steps detailed in this document. For each step of the workshop, since Karpenter is built in the open source, and has a lot of cloud agnostic elements, there is a link to the AWS version (in the step's header), as much of the AWS workshop will translate directly over to Azure. However, for any steps where it diviates, there will be a few important notes, and adjustments detailed under those steps.
+
+## Basic Cheet Sheet
+
+When you see `aws-node-viewer` use `aks-node-viewer` instead.
```

Comment:
```suggestion
When you see `eks-node-viewer` use `aks-node-viewer` instead.
```
(And probably no need to repeat this for every step?)

Reply:
Yea, probably don't need to repeat, but felt weird if I left the steps blank/empty, but also a little weird if I said "no required adjustments" while you do have to swap the node viewers.